### DISCUSSION TITLE
Bundle parent product variant price not included in bundle price?

### DISCUSSION THREAD

**Discussion by jakeclifford** [Shopify/function-examples#487] [Feedback: Cart Transform API]
I want to add a base price to a bundle so was hoping that the price of the parent product would be included when using the cartTransform merge operation, is there a way to add this price to the final price?

**Comment by sillycube** [comment]
You can only store the base prices of all bundles into a metafield and retrieve this meta in the merge function. Line item property is also possible but not secure as meta

**Reply by huykon** [reply]
But in cart line items, we don't store metafield of parent product
